src/formatters/functions.rs

```diff
@@ -464,7 +470,8 @@ pub fn format_function_args(
         }
 
         FunctionArgs::String(token_reference) => {
-            if ctx.config().no_call_parentheses
+            if (ctx.config().no_call_parentheses
+                || ctx.config().call_parentheses == CallParenType::None)
```

Comment:
It might be nice to abstract out these condition checks onto a method implemented on config, so instead we can just call, e.g. something like
`ctx.config().should_omit_string_parens()` -> no_call_parentheses = true || CallParenType == None
`ctx.config().should_omit_table_parens()` -> no_call_parentheses = true || CallParenType == NoTable || CallParenType == None

Would help in future if we ever decide to add e.g. NoString or something